tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -841,6 +841,10 @@ func getURLBase(host string, path string) (*url.URL, error)  {
         return nil, whiskErr
     }
 
+    if !strings.HasPrefix(host, "http") {
```

Comment:
Hmm, what if the string begins with `https://`, won't `https://` be prepended again?